plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -296,6 +297,19 @@ public void dropSchema(ConnectorSession session, String schemaName)
         throw new TrinoException(NOT_SUPPORTED, "This connector does not support dropping schemas");
     }
 
+    @Override
+    protected void dropTable(ConnectorSession session, RemoteTableName remoteTableName)
```

Comment:
I do not matching by name. It looks like you could pass the `boolean temporaryTable` flag explicitly via parameter.